src/pipe_wrap.cc

```diff
@@ -76,9 +76,11 @@ void PipeWrap::Initialize(Local<Object> target,
   env->set_pipe_constructor_template(t);
 
   // Create FunctionTemplate for PipeConnectWrap.
-  Local<FunctionTemplate> cwt =
-      FunctionTemplate::New(env->isolate(),
-      [](const FunctionCallbackInfo<Value>& a){CHECK(a.IsConstructCall());});
+  auto constructor = [](const FunctionCallbackInfo<Value>& args) {
+      CHECK(args.IsConstructCall());
```

Comment:
Just when I thought I got it :) I'll fix it really try to be more careful in future.